tables.md

```diff
@@ -0,0 +1,204 @@
+
+There are additional descriptors for human readability that you can add to your column sections. These include:
+
+- `suppressOutput`  If true, suppresses any output that would be generated when converting cells in this column. The value of this property becomes the suppress output annotation for the described column. The default is false.(from w.3/org-tabular-metadata)
+-  `propertyuri`, `valueurl`, `aboutUrl`  This metadata creates a relationship model between data in each column by different combinations of aboutUrl, propertyUrl, and valueUrl on existing columns, and defining new virtual columns to supply additional information. (from w.3/org-tabular-metadata)
+- `label` this can be used for additional context.
+
+
+## Table Breakdown
+
+This section will help you how to write your table section of your JSON-LD file. But also, how to describe each column from your csv file. 
+
+The first section will show you what needs to be done for each csv you are using.
+
+The second section will go through a step by step process of writing and describing the columns in your csv.
+
+## First Section
+
+```JSON
+"tables": [
+    {
+        "url": "football.csv",
+        "tableSchema": {
+            "columns": [
+```                
+                
+In order to write a JSON-LD to include mulitple csvs. You need to include the csvs column details within a tables [ ] section. 
+
+`url` This link property gives the single URL of the CSV file that the table is held in, relative to the location of the metadata document. (from w.3/org-tabular-metadata)
+
+`tableSchema` An object property that provides a single schema description, used as the default for all the tables in the group. This may be provided as an embedded object within the JSON metadata or as a URL reference to a separate JSON object that is a schema description.(from w.3/org-tabular-metadata)
+
+`columns` This will be the area where you describe each column from your csv.
+
+## Second section
+
+The way you write your JSON-LD should follow the same layout as your csv.
+
+You write the JSON-LD runs from top to bottom and you should read your csv from left to right.
+
+For example the first entry in the JSON-LD should be the column that appears on the left hand side of the csv.
+
+### Time Period
+
+```JSON              
+                {
+                    "title": "Period Type",
+                    "name": "period_type",
+                    "data_type": "string",
+                    "suppressOutput": true
+                },
+```  
+This section will look at the period type column. This entry should include a `title`, `name` and `data type`. The title provies the human readable name of the column. The name provides the machine readable name of the column. This should be in `snake_case`. The data type provides the detail of what data type is being represented in the column. The suppress output uses the information you have provided.
```

Comment:
`name` links the column definition in JSON to the csv column. We don't need to talk about the name format of the csv column header as this is covered in the csv document.

Reply:
Updated.